lib/Doctrine/DBAL/Types/TypeRegistry.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\DBALException;
+use function array_search;
+use function in_array;
+
+/**
+ * Type registry is responsible for holding holding a map of known DBAL types.
+ * Type instances are implemented as flyweight pattern.
+ *
+ * @internal TypeRegistry class exists for forward compatibility, its API should not be considered stable.
```

Comment:
```suggestion
 * @internal the TypeRegistry class exists for forward compatibility, its API should not be considered stable.
```

Reply:
IMO not, TypeRegistry is a name.

Reply:
"TypeRegistry exists" would be correct because of this, but "TypeRegistry class exists" isn't I think.